package/ci/python3-entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -xe
 CMD=${1:- -m pytest}
 LOGFILE=${2:- pytest.log}
+DE_BRANCH=${3:-master}
```

Comment:
Maybe the default should be 1.6?

Reply:
Yes, this makes things more consistent.
I just pushed the update.